Src/Fido2.Models/AuthenticatorAttestationRawResponse.cs

```diff
@@ -15,7 +15,7 @@ public sealed class AuthenticatorAttestationRawResponse
     public byte[] RawId { get; set; }
 
     [JsonPropertyName("type")]
-    public PublicKeyCredentialType Type { get; set; } = PublicKeyCredentialType.PublicKey;
+    public PublicKeyCredentialType? Type { get; set; }
```

Comment:
The type doesn't seem to be optional in the spec: https://w3c.github.io/webappsec-credential-management/#the-credential-interface.

Reply:
I will check the docs tomorrow. I assumed the test tool is based on the up-to-date requirements.

Reply:
I have checked it.
The test case is about that exactly, it must be in the request:
"F-4 Send ServerAuthenticatorAttestationResponse that is missing "type" field and check that server returns an error"
If it's missing then the server must return an error, we can't give it a default value and go on.

Reply:
How should we check for missing "type" in the request?
These lines in your code are checking that it must be PublicKey:
```
        if (Raw.Type != PublicKeyCredentialType.PublicKey)
            throw new Fido2VerificationException(Fido2ErrorCode.InvalidAttestationResponse, 
        Fido2ErrorMessages.AttestationResponseNotPublicKey);
```
I guess that null is as bad as any other non-publickey value and we should check that.

Reply:
Yeah, for now, it is has to be publickey, anything other than that should result in error.

Reply:
Then the original code (= PublicKeyCredentialType.PublicKey;) hides an error if the input is null.
Which breaks the conformance testcase mentioned above.